config.go

```diff
@@ -600,6 +602,16 @@ func loadConfig() (*config, []string, error) {
 		return nil, nil, err
 	}
 
+	// Limit the max orphan count to a sane vlue.
+	if cfg.MaxOrphanTxs < 0 {
```

Comment:
I'd suggest making the config option a `uint32` so the parser will prevent negatives for you.